src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/service/VisitSchedulerService.kt

```diff
@@ -144,16 +144,31 @@ class VisitSchedulerService(
     return visitSchedulerClient.getVisitSessions(prisonCode, prisonerId, min, max)
   }
 
-  fun getAvailableVisitSessions(prisonCode: String, prisonerId: String, requestedSessionRestriction: SessionRestriction?): List<AvailableVisitSessionDto>? {
-    val dataRange = prisonService.getToDaysBookableDateRange(prisonCode)
-    val restrictions = prisonerProfileService.getRestrictions(prisonerId)
-    val sessionRestriction = if (hasClosedRestriction(restrictions)) CLOSED else requestedSessionRestriction ?: OPEN
+  fun getAvailableVisitSessions(prisonCode: String, prisonerId: String, requestedSessionRestriction: SessionRestriction?, visitors: List<Long>?): List<AvailableVisitSessionDto>? {
```

Comment:
should we return Not Null from this method and force it to return an emptyList?

Reply:
Good idea,.. updated code